vars/Debian.yml

```diff
@@ -1,2 +1,3 @@
 mysql_daemon: mysql
-mysql_hardening_mysql_conf: '/etc/mysql/my.cnf'
+mysql_hardening_mysql_conf_file: '/etc/mysql/my.cnf'
+mysql_hardening_mysql_conf_dir: '/etc/mysql/conf.d'
```

Comment:
Shouldn't this be `mysql_hardening_mysql_confd_dir` (with a `d` after `conf`) since its the `conf.d` directory?

`/etc/mysql/` is the `mysql_hardening_mysql_conf_dir`.